pages/views.py

```
@@ -1,4 +1,5 @@
 from django.views.generic import TemplateView
+from django.shortcuts import render
```

Ruff (0.11.9)

`django.shortcuts.render` imported but unused

Remove unused import: `django.shortcuts.render`

(F401)